core/commands/config.go

```diff
@@ -132,6 +113,23 @@ Set the value of the 'Datastore.Path' key:
 			} else {
 				output, err = setConfig(r, key, value)
 			}
+			if err != nil {
+				return err
+			}
+			switch strings.ToLower(output.Key) {
+			case "ui.wallet.initialized":
```

Comment:
output.Key may not be this if set through json for the whole `config.UI` section?